scripts/get-helm-3

```diff
@@ -17,8 +17,7 @@
 # The install script is based off of the MIT-licensed script from glide,
 # the package manager for Go: https://github.com/Masterminds/glide.sh/blob/master/get
 
-PROJECT_NAME="helm"
-
+: ${PROJECT_NAME:="helm"}
```

Comment:
PROJECT_NAME, maybe better as HELM_EXE? or something?

Reply:
Good idea. Or BINARY_NAME etc, something that conveys the purpose

Reply:
Would be nice to make this a command line option as well:

```bash
    '--name'|-n)
       shift
       if [[ $# -ne 0 ]]; then
           BINARY_NAME="${1}"
       else
           echo -e "Please provide the desired binary name. e.g. --name helm3 or -v helm"
       fi
       ;;
```

and for the help function:

```bash
  echo -e "\t[--name|-n <binary_name>] . When not defined defaults to helm otherwise useful for installing side by side a previous version"
  echo -e "\te.g. --name helm3 or -n helm3"
```